.gitignore

```diff
@@ -46,3 +46,7 @@ rpm/src/
 
```

Comment:
Fixed in feature/970_create_entity   (issue number for STRING/NUMBER/BOOL missing)